packages/apps-config/src/api/params/inflation.ts

```diff
@@ -16,7 +16,7 @@ const DEFAULT_PARAMS: InflationParams = {
   falloff: 0.05,
   idealStake: 0.5,
   maxInflation: 0.1,
-  minInflation: 0.025
+  minInflation: 0
```

Comment:
As far as we set inflation to `0` in Node's code we need to be sure that it will be the same in the BlockViewer until we switch it back.

All these parameters are using to calculate Inflation on the fly. By setting `minInflation` to `0` you just allow it to be `0`. It is not enough for us. The easiest solution, for now, is to use 0% always.

Reply:
I've hardcoded an [empty](https://github.com/Cerebellum-Network/block-viewer/blob/5cdb938e5eed3150349bdd811dcf1b330a66bb73/packages/react-hooks/src/useInflation.ts#L16) value for the [inflation](https://github.com/Cerebellum-Network/block-viewer/blob/5cdb938e5eed3150349bdd811dcf1b330a66bb73/packages/page-staking/src/useSortedTargets.ts#L176) object: `{ inflation: 0, stakedReturn: 0 }`